edx-platform/pearson-theme/lms/templates/index.html

```diff
@@ -85,3 +111,56 @@ <h1 class="title">
   $(window).load(function() {$('#signup_action').trigger("click");});
 </script>
 % endif
+
+%if search_courses_home:
+  <script type="text/javascript">
+    var searchHome = (function() {
+      'use strict';
+
+      var inputField, coursesList, filters, iconClear, valueInput;
+
+      function initElements() {
+        inputField = $("#search-input");
+        coursesList = $(".courses-listing li");
+        filters = $(".quick-filter-item");
+        iconClear = $(".clear-search-icon");
+      }
+
+      function eventSearch() {
+        inputField.on("input", function(){
+          valueInput = $(this).val().toLowerCase();
+          coursesList.filter(function(){
+           $(this).toggle($(this).text().toLowerCase().indexOf(valueInput) > -1)
```

Comment:
Why are you using this toggle? @amalbas

Reply:
Missing semicolon. @amalbas

Reply:
@Squirrel18 toggle hide the courses that don't match with the value on the input search.

Reply:
Could you extract "$(this).text().toLowerCase()" into a variable definition, it's just to have more comfortable code reading. @amalbas

Reply:
@Squirrel18 done